app/Controller/PagesController.php

```diff
@@ -28,54 +28,72 @@
  * @package       app.Controller
  * @link http://book.cakephp.org/2.0/en/controllers/pages-controller.html
  */
-class PagesController extends AppController {
+class PagesController extends AppController
+{
 
-/**
+    /**
  * This controller does not use a model
  *
  * @var array
  */
-	public $uses = array();
-	public $uses2 = array();
-	public $uses3 = array();
+    public $uses = array();
```

Comment:
__[staging][code_sniffer]__
* Equals sign not aligned with surrounding assignments; expected 2 spaces but found 1 space